user-interface/src/main/java/life/qbic/datamanager/views/account/AddPersonalAccessTokenDialog.java

```diff
@@ -31,12 +28,11 @@
  */
 public class AddPersonalAccessTokenDialog extends DialogWindow {
 
-  private static final Logger log = logger(AddPersonalAccessTokenDialog.class);
   @Serial
   private static final long serialVersionUID = 6149070385320827888L;
   private final TextField tokenDescription = new TextField();
   private final Select<Duration> expirationDate = new Select<>();
-  private final Binder<PersonalAccessTokenDTO> personalAccessTokenDTOBinder;
+  private final Binder<PersonalAccessTokenFrontendBean> personalAccessTokenDTOBinder;
 
```

Comment:
NIT: While i like the rename of the object i'd suggest to also rename the fieldName as well